ROADMAP.md

```diff
@@ -290,6 +287,25 @@ the [libp2p test-plans repository].
 
 - [First proof of concept](https://github.com/libp2p/test-plans/pull/20)
 
+### WebRTC
+
+**Status**: In progress
+
+**What?** WebRTC is a transport protocol supported by all major browsers. Those
+browsers allow the establishment of connections to remote endpoints that don't
+have a TLS certificate signed by a trusted certificate authority. In addition
+WebRTC includes hole punching capabilities.
+
+**Why?** In most p2p networks the majority of nodes does not have a signed TLS
+certificate. With WebRTC browsers will thus be able to connect to these
+previously unreachable nodes. In addition, being able to hole punch allows
```

Comment:
This should probably reference WebTransport.

Reply:
Good idea. Done with https://github.com/libp2p/specs/pull/414/commits/df4fb0cc85f6f30079b4c5c0c72a8b354ead2a20.